src/utilities/modules/dispensary/pay.js

```diff
@@ -80,5 +89,9 @@ export const pay = (currentUser, patient, script, cashAmount, scriptTotal) => {
   if (usingCredit && creditToAllocate > 0) throw new Error('Credit not fully allocated');
 
   // Ensure the script is finalised once paid.
+  script.subtotal = subtotal;
+  script.total = subtotal;
```

Comment:
just a question: should the total be the subtotal - discount amount?

Kind of related, there's now this: https://github.com/sussol/msupply/issues/5085

Reply:
Yeah - at this point I have no idea what implementation to do. Same with https://github.com/sussol/msupply/issues/5086 :(